test/testautomation_blit.c

```diff
@@ -172,12 +178,12 @@ int blit_testRandomToRandomSVGAMultipleIterations(void *arg) {
         SDL_free(buf);
     }
     // Check result
-    Uint32 hash = hashSurfacePixels(dest_surface);
+    const unsigned long hash = hashSurfacePixels(dest_surface);
     // Clean up
     SDL_SaveBMP(dest_surface, "250k_scalar.bmp");
     SDL_DestroySurface(dest_surface);
     SDLTest_AssertCheck(hash == scalar_hash || hash == x86_simd_hash,
-                        "Should render identically, expected 0x%x (scalar) or 0x%x (x86_simd), got 0x%x",
+                        "Should render identically, expected 0x%lx (scalar) or 0x%lx (x86_simd), got 0x%lx",
```

Comment:
Not sure why you switched away from SDL data types, but you can use `SDL_PRIu32` in the format string.

Reply:
Was trying to fix the build on some platforms and didn't know about those helpers. Thanks! Back to SDL data types.